docs/read-models.md

```
@@ -54,21 +55,22 @@ repo.read_models::<GameView>().upsert(&updated_view)?;
 
 ## 1. Eventually Consistent (The Default Path)
 
-This is the bread-and-butter pattern for CQRS: events are published through
-an outbox, and a separate denormalizer/projector process consumes them to
-update read models.
+This is the bread-and-butter pattern for CQRS: aggregates record replayable
+event records for their own state, domain or integration messages are published
+through an outbox, and a separate denormalizer/projector process consumes those
+messages to update read models.
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)